Source/WebCore/html/OffscreenCanvas.cpp

```diff
@@ -524,10 +524,10 @@ void OffscreenCanvas::scheduleCommitToPlaceholderCanvas()
     }
 }
 
-void OffscreenCanvas::createImageBuffer() const
+RefPtr<ImageBuffer> OffscreenCanvas::createImageBuffer() const
 {
     m_hasCreatedImageBuffer = true;
```

Comment:
Now that createImageBuffer doesn't associate a buffer with this OffscreenCanvas, it would make more sense to have an `m_hasSetImageBuffer` member instead of `m_hasCreatedImageBuffer` and to set this in setImageBuffer. There's the situation now (I think) where you could call `transferToImageBitmap` before a buffer is created, then `m_hasCreatedImageBuffer` will be set to `true`, but no image buffer would have been set.